ui/webui/src/components/storage/CustomMountPoint.jsx

```diff
@@ -0,0 +1,262 @@
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useState, useEffect } from "react";
+
+import {
+    Alert,
+    Checkbox,
+    Flex,
+    HelperText,
+    HelperTextItem,
+    Popover,
+    Select,
+    SelectOption,
+    SelectVariant,
+    TextContent,
+} from "@patternfly/react-core";
+import { HelpIcon } from "@patternfly/react-icons";
+
+import { ListingTable } from "cockpit-components-table.jsx";
+
+import cockpit from "cockpit";
+
+import { AnacondaPage } from "../AnacondaPage.jsx";
+
+import {
+    gatherRequests,
+    setManualPartitioningRequests,
+    findPartitioning,
+} from "../../apis/storage.js";
+
+import "./CustomMountPoint.scss";
+
+const _ = cockpit.gettext;
+
+const MountPointSelect = ({ partition, requests, mountpoint, handleOnSelect, isDisabled }) => {
+    // TODO: extend?
+    const defaultOptions = [
+        { value: "/", name: "root" },
+        { value: "/boot", name: "boot" },
+        { value: "/home", name: "home" },
+    ];
+
+    const duplicatedMountPoint = mountpoint => {
+        return requests.filter(r => r["mount-point"] === mountpoint).length > 1;
+    };
+
+    const [isOpen, setIsOpen] = useState(false);
+    // Filter selected
+    const options = defaultOptions.filter(val => val.value !== mountpoint);
+    if (mountpoint !== "") {
+        options.push({ value: mountpoint });
+    }
+
+    return (
+        <Flex direction={{ default: "column" }} spaceItems={{ default: "spaceItemsNone" }}>
+            <Select
+              variant={SelectVariant.typeahead}
+              className="mountpoint-select"
+              typeAheadAriaLabel={_("Select a mount point")}
+              selections={mountpoint || null}
+              isOpen={isOpen}
+              onToggle={isOpen => setIsOpen(isOpen)}
+              onSelect={(_evt, selection, _) => { setIsOpen(false); handleOnSelect(selection, partition) }}
+              isDisabled={isDisabled}
+              isCreatable
+              shouldResetOnSelect
+            >
+                {options.map((option, index) => (
+                    <SelectOption
+                      key={index}
+                      className={`select-option-${option.name}`}
+                      value={option.value}
+                    />
+                ))}
+            </Select>
+            {mountpoint !== "" && duplicatedMountPoint(mountpoint) &&
+                <HelperText component="ul">
```

Comment:
Please remove component="ul". Why you want it as list here? Probably you copied this from a place with many helper messages.